fdk/context.py

```diff
@@ -125,6 +145,114 @@ def __is_gateway(self):
                 == constants.INTENT_HTTP_REQUEST)
 
 
+class TracingContext(object):
+
+    def __init__(self, is_tracing_enabled, trace_collector_url,
+                 trace_id, span_id, parent_span_id,
+                 is_sampled, flags):
+        """
+        Tracing context here to be a placeholder
+        for tracing-specific attributes
+        :param is_tracing_enabled: tracing enabled flag
+        :type is_tracing_enabled: bool
+        :param trace_collector_url: APM Trace Collector Endpoint URL
+        :type trace_collector_url: str
+        :param trace_id: Trace ID
+        :type trace_id: str
+        :param span_id: Span ID
+        :type span_id: str
+        :param parent_span_id: Parent Span ID
+        :type parent_span_id: str
+        :param is_sampled: Boolean for emmitting spans
+        :type is_sampled: int (0 or 1)
+        :param flags: Debug flags
+        :type flags: int (0 or 1)
```

Comment:
This is actually a string because the Zipkin implementation may change in the future - I don't think it causes issues but the docs could have a note of it here.